src/main/java/billgates/database/MySQLDatabaseGateway.java

```diff
@@ -171,18 +183,42 @@ public void insertEntry(int billId, QueryEntryData entry) {
             Statement statement = connection.createStatement();
 
             String query = String.format("""
-                            INSERT INTO bill%d (value, date, currency, description, `from`, `to`, location) VALUE (
-                            %f, "%s", "%s", "%s", "%s", "%s", "%s"
+                            INSERT INTO bill%d (entry_id, value, date, currency, description, `from`, `to`, location, split_bill_id) VALUE (
```

Comment:
Do we need `entry_id` here? 

Remember that `entry_id` is auto-incremented.

Reply:
This is for testing and easier to track the entry that we just inserted. 

For example, if we inserted like 3 entries of eating at Burger King, how do we know which entry ID correspond to which entry (Without the GUI Display)? Because our only way to keep track of the entry is through the ID. 

My test case is written based on this, as I insert a testEntry with a specified ID, so that I will be able to track changes on this testEntry through the ID.